libs/platform/src/lib/components/form/base.input.ts

```diff
@@ -209,6 +209,15 @@ export abstract class BaseInput extends BaseComponent
 
     /** @hidden */
     ngAfterViewInit(): void {
+
+        if (this.ngControl) {
```

Comment:
Why do we need this here? Is this because we are using our control outside of FormField? The idea was always to use FormField that controls everything, this way there will be too much activity going on. 

If yes, then we should turn this on only and if input is used as standalone.  Or what are the usecaes for this ?